management/log_test.go

```diff
@@ -3,49 +3,26 @@ package management
 import (
 	"testing"
 
-	"github.com/auth0/go-auth0"
+	"github.com/stretchr/testify/assert"
 )
 
-func TestLog(t *testing.T) {
-	// Save a log from the listing so we can look it up by ID later
-	var firstLog *Log
+func TestLogManager(t *testing.T) {
+	// Limit results to 5 entries, starting from the first page.
+	logs, err := m.Log.List(Page(1), PerPage(5))
+	assert.NoError(t, err)
+	assert.Greater(t, len(logs), 0, "can't seem to find any logs, have you ran any tests before?")
 
-	t.Run("List", func(t *testing.T) {
-		// Limit results to 5 entries, starting from the first page
-		logs, err := m.Log.List(Page(1), PerPage(5))
-		if err != nil {
-			t.Error(err)
-		}
-
-		for i, log := range logs {
-			t.Logf("%v\n", log)
-
-			// Save the first log for reading later
-			if i == 0 {
-				firstLog = log
-			}
-		}
-	})
+	actualLog, err := m.Log.Read(logs[0].GetID())
+	assert.NoError(t, err)
+	assert.Equal(t, logs[0], actualLog)
 
-	t.Run("Read", func(t *testing.T) {
-		log, err := m.Log.Read(auth0.StringValue(firstLog.ID))
-		if err != nil {
-			t.Error(err)
-		}
-		t.Logf("%v\n", log)
-	})
+	// Search by type "Success Exchange" and limit results to 5 entries
+	logs, err = m.Log.List(Parameter("q", `type:"seacft"`), PerPage(5))
```

Comment:
If we use `seacft`, wouldn't that require using a tenant with login events?
This SDK is management-only at the moment, so these tests could not be pointed at a fresh tenant because there would be an expectation of events that can't be generated by this SDK alone.

What about `sapi` ("Successful management API write event") which represents a Management API operation, and thus could be satisfied with any previous tests from this SDK?

Reply:
Makes total sense! Thanks for spotting this! Fixed in: [ce0a037](https://github.com/auth0/go-auth0/pull/61/commits/ce0a03735346b7f938be28e2e0a1d93c8abe83c2)